WalletWasabi/WabiSabi/Client/AmountDecomposer.cs

```diff
@@ -361,7 +364,7 @@ private static Money FindChange(IEnumerable<Output> decomposition, HashSet<Outpu
 					?? denoms.FirstOrDefault(x => x.EffectiveCost <= remaining);
 
 				// We can only let this go forward if at least 2 outputs can be added (denom + potential change)
-				if (denom is null || remaining < MinAllowedOutputAmount + ChangeFee || remainingVsize < denom.ScriptType.EstimateOutputVsize() + ChangeScriptType.EstimateOutputVsize())
```

Comment:
These following lines in naive and pre decompositions have the same rationale as https://github.com/zkSNACKs/WalletWasabi/pull/10824/files#r1212766658